### PR TITLE
Added command to extract FormIO messages at best effort.

### PR DESCRIPTION
**Pull request by joeribekker**
The number of "missing translations" reports got a bit annoying. I added this command but I remember @SilviaAmAm had something similar but could not find the script.

This command adds all new translations to the file. I added those that got reports.

You can just run:

```
python src/manage.py formio_makemessages --dryrun --report-unknown --report-output
```

...to see it in action without any damage.

### CODE REVIEW

File: src/openforms/formio/management/commands/formio_makemessages.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        parser.add_argument(
+            "--formio",
+            action="append",
+            help="Point to a specific folder where FormIO components are located.",
+        )
+        parser.add_argument(
+            "--dryrun",
+            "--dry-run",
+            action="store_true",
+            help="Do not write to translations back to file.",
```
Comment (sergei-maertens) [inline src/openforms/formio/management/commands/formio_makemessages.py:39]:
```suggestion
            help="Do not write translations back to file.",
```

File: src/openforms/formio/management/commands/formio_makemessages.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    "label: '([^']+)'",
+    "\.t\('([^']+)'[\),]",
+    "tooltip: '([^']+)'",
+    "errorMessage\('([^']+)'[\),]",
+]
+
+
+class Command(BaseCommand):
+    help = (
+        "Best effort to extract messages from FormIO javascripts. "
```
Comment (sergei-maertens) [inline src/openforms/formio/management/commands/formio_makemessages.py:19]:
```suggestion
        "Best effort to extract messages from FormIO javascript source code. "
```

File: src/openforms/formio/management/commands/formio_makemessages.py [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+        # Update with new and existing translations (new will go at the bottom)
+        result.update(found_translations)
+
+        # Overwrite the existing translation keys with their existing
+        # translation values
+        result.update(existing_translations)
+
+        if report_output:
+            self.stdout.write(json.dumps(result, indent=2))
```
Comment (sergei-maertens) [inline src/openforms/formio/management/commands/formio_makemessages.py:147]: I think you can use `json.dump(result, self.stdout, indent=2)` too here, but I'm not sure

**Review by sergei-maertens — CHANGES_REQUESTED** [review]

File: src/openforms/formio/management/commands/formio_makemessages.py (hunk 3 above)
Comment (joeribekker) [reply src/openforms/formio/management/commands/formio_makemessages.py:147]: Just tested it. It works, but formatting is barfed. Almost every word is put on a new line.

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**